### PR TITLE
Document that Submariner is a CNCF sandbox project

### PR DESCRIPTION
**Pull request by dfarrell07**
Now that Submariner is a CNCF sandbox project, it is required by the
CNCF that we list this status prominently and use this wording.

https://github.com/cncf/foundation/blob/master/website-guidelines.md

Signed-off-by: Daniel Farrell <dfarrell@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ provider, as it utilizes off-the-shelf components to establish encrypted tunnels
 Note that Submariner is in the **pre-alpha** stage, and should not be used for production purposes. While we welcome usage and
 experimentation, it is quite possible that you could run into bugs.
 
+Submariner is a Cloud Native Computing Foundation sandbox project.
```
Comment (nyechiel) [inline README.md:34]:
```suggestion
Submariner is a Cloud Native Computing Foundation (CNCF) Sandbox project.
```

**Review by nyechiel — COMMENTED** [review]
One suggestion. Also, @dfarrell07 should we mention it in the website home page?

**Comment by nyechiel** [comment]
> One suggestion. Also, @dfarrell07 should we mention it in the website home page?

Oh, I found https://github.com/submariner-io/submariner-website/pull/517/files, sorry :)

File: README.md (hunk 1 above)
Comment (skitt) [reply README.md:34]: The sentence specifically has to be “We are a Cloud Native Computing Foundation sandbox project.” or a variant thereof; we can’t add “(CNCF)” or capitalise “sandbox”. See https://github.com/cncf/foundation/blob/master/website-guidelines.md

**Review by skitt — CHANGES_REQUESTED** [review]
Could you add a pointer to https://github.com/cncf/foundation/blob/master/website-guidelines.md in the commit message?

File: README.md (hunk 1 above)
Comment (nyechiel) [reply README.md:34]: Thanks for the link. I still find it odd that "sandbox" is not capitalised (it is in many other paces in the CNCF website), but these guidelines are very clear.

Comment (dfarrell07) [reply README.md:34]: Yeah it's weird, in the legal-focused docs it's not capitalized but everywhere else it is.

**Review by skitt — APPROVED** [review]

**Review by nyechiel — APPROVED** [review]

**Pull request merged**